pkg/cmd/cli/backuplocation/create.go

```diff
@@ -55,14 +55,15 @@ func NewCreateCommand(f client.Factory, use string) *cobra.Command {
 }
 
 type CreateOptions struct {
-	Name             string
-	Provider         string
-	Bucket           string
-	Prefix           string
-	BackupSyncPeriod time.Duration
-	Config           flag.Map
-	Labels           flag.Map
-	AccessMode       *flag.Enum
+	Name                  string
+	Provider              string
+	Bucket                string
+	Prefix                string
+	BackupSyncPeriod      time.Duration
+	StoreValidationPeriod time.Duration
```

Comment:
would `validationFrequency` be a better name for this field?

Reply:
Yes, yes indeed!